### PR TITLE
Proposals should clearly state business factors and user risks

### PR DESCRIPTION
**Pull request by AramZS**
Split from #46

### CODE REVIEW

**Comment by npdoty** [comment]
Does "business factors" refer to use cases, like what functionality is provided by any particular specification?
Does "user risk" refer to privacy and security implications? 

There is a paragraph above this one that requires documenting privacy and security implications.

I'm not sure I've ever seen it listed as a separate requirement in a charter, but I think W3C specifications do typically describe use cases, or, more often, that use cases are detailed in other group deliverables. Some charters note that non-normative use case and requirements documents may be published by the group in the course of their work.

**Comment by ShivanKaul** [comment]
I think we should disambiguate "business factors" as well, but in favour of having some text along these lines.

File: charter.html [hunk 1]
```diff
@@ -281,6 +281,8 @@ <h2>Success Criteria</h2>
           aspects of normative specifications developed by the WG.
 
     <p>Normative specifications which have user-facing features should contain a section on accessibility that describes the benefits and impacts, including ways specification features can be used to address them, and recommendations for maximizing accessibility in implementations.</p>
+          
+    <p>In considering features, the group will state, during the development of those features or in the text of the document, what functionality is aimed to be provided and what specific risks it aims to address. Proposals should clearly state business factors and user risks they will consider and how they will decrease such risks and what safeguards will be put in place to decrease such risk and continue to enable the business factors they wish to preserve.</p>
```
Comment (martinthomson) [inline charter.html:285]: Throwing out a different framing:

> Many of the specifications that this group is expected to produce will have a special need to balance privacy risks with the interests of businesses that rely on advertising.  Specifications produced by the group shall contain clear justification for any use of private information or any privacy risks that are introduced.

Comment (martinthomson) [reply charter.html:285]: > This group will need to balance privacy risks with the interests of businesses that rely on advertising.  Normative specifications produced by the group shall contain clear justification for any use of private information or any privacy risks that are introduced.

Comment (benjaminsavage) [reply charter.html:285]: As the specifications produced by this group are expected to strike an appropriate balance between the interests of businesses and user privacy, specifications will discuss these tradeoffs and provide rationales way these factors are balanced.

Comment (eriktaubeneck) [reply charter.html:285]: Does "clear justification" have a well defined meaning?

Comment (mnot) [reply charter.html:285]: Not _shall_ -- make it aspirational.

Comment (eriktaubeneck) [reply charter.html:285]: "risks that are introduced" implies there is a baseline.

Comment (martinthomson) [reply charter.html:285]: In the interest of having more options...

> This group will need to balance privacy risks with the interests of businesses that rely on advertising. Normative specifications produced by the group should discuss any privacy tradeoffs and provide rationale for any use or release of private information.

Comment (AramZS) [reply charter.html:285]: Perhaps (from discussion in group):

> This group will need to balance privacy risks with the interests of businesses that rely on advertising. Any balance should give greater weight to user privacy over advertising business objectives, in the spirit of the [priority of constituencies](https://www.w3.org/TR/design-principles/#priority-of-constituencies). Normative specifications produced by the group should discuss any privacy tradeoffs and provide rationale for any use or release of private information.

Comment (martinthomson) [reply charter.html:285]: You know, after discussing this, I've come full circle.

> This group will need to balance privacy risks with the interests of businesses that rely on advertising.

This balance is central to the work we're doing, and acknowledging that is important.  It's just framing for the next piece...

> Normative specifications produced by the group should discuss any privacy tradeoffs and provide rationale for any use or release of private information.

This is the key success criterion here.  I think that it is OK.

**Comment by AramZS** [comment]
We will return to this Thursday. Please place any changes before then.

**Comment by martinthomson** [comment]
I would be OK with a formalization of Nick's proposed list.  Or the text I proposed.

**Comment by ShivanKaul** [comment]
@npdoty Points 1 & 2 from your list seem fine, but I'm not sure why we need 3.

**Pull request merged**